src/api/entities/Instruction/index.ts

```diff
@@ -474,8 +474,8 @@ export class Instruction extends Entity<UniqueIdentifiers, string> {
       return {
         status: middlewareInstructionStatusToInstructionStatus(status),
         createdAt: new Date(createdBlock!.datetime),
-        tradeDate: new Date(tradeDate),
```

Comment:
does this need a test case for branch coverage?

As long as sonar is fine with it its OK to merge